eng/common/scripts/Add-Issue-Comment.ps1

```diff
@@ -38,16 +33,12 @@ if (!$commentPrefixValue) { $commentPrefixValue = $CommentPrefix }
 if (!$commentValue) { $commentValue = $Comment }
 if (!$commentPostFixValue) { $commentPostFixValue = $CommentPostFix }
 
-$PRComment = "$commentPrefixValue $commentValue $commentPostFixValue"
-
-$data = @{
-  body = $PRComment
-}
-
 try {
-  $resp = Invoke-RestMethod -Method POST -Headers $headers -Uri $apiUrl -Body ($data | ConvertTo-Json)
+  $resp = AddIssueComment -RepoOwner $RepoOwner -RepoName $RepoName `
+          -IssueNumber $IssueNumber -CommentPrefix $commentPrefixValue `
+          -Comment $commentValue -CommentSuffix $commentPostFixValue
 }
 catch {
-  LogError "Invoke-RestMethod [ $apiUrl ] failed with exception:`n$_"
+  LogError "AddIssueComment failed with exception:`n$_"
```

Comment:
If we are going to provide helpers for invoking github apis we should include the error handling there as well

Reply:
Am not sure how to properly do that. Should I use the try catch inside the `Invoke-GitHub-API.ps1` script. if so should I also use the `logging.ps1` to log the errors there? Won't that cause DevOps to exit prematurely?

Reply:
That is a fair point perhaps we should let the error just flow and if someone wishes to handle it then can do it correctly in their context.